src/Sylius/Behat/Context/Setup/ProductReviewContext.php

```diff
@@ -75,16 +75,17 @@ public function productHasAReview(ProductInterface $product)
     }
 
     /**
-     * @Given /^(this product) has(?:| also) a review titled "([^"]+)" and rated (\d+) added by (customer "[^"]+")(?:|, created (\d+) days ago)$/
+     * @Given /^(this product) has(?:| also) a review titled "([^"]+)" and rated (\d+)(?:| with a comment "([^"]+)") added by (customer "[^"]+")(?:|, created (\d+) days ago)$/
```

Comment:
`This product has a review title "Foo" and rated 3 added by customer "Bar"` won't work, too many optional placeholders.